server/dbs/postgres_sqlalchemy.py

```diff
@@ -84,6 +83,10 @@ def _isFunctionAllowed(self, funcname):
                 if (func[1] not in ('i', 's') and
                         self._allowedFunctions.get(func[0], False)):
                     self._allowedFunctions[func[0]] = False
+            # Whitelist some functions if they exist
+            for func in funcs:
+                if func[0] in ('pg_sleep', ):
+                    self._allowedFunctions[func[0]] = True
```

Comment:
Good point.  I'll change things so it is only whitelisted in the tests, not in general.